gapic-generator/templates/default/service/client/method/docs/sample_response/_loop.erb

```diff
@@ -0,0 +1,14 @@
+<%- assert_locals values -%>
+<%= values["collection"].gsub("$resp", "response") %>.each do |result|
+  <%- values["body"].each do |b| -%>
+  <%- if b.key? "comment" -%>
```

Comment:
We could probably reuse the `service/client/method/docs/sample_response/_comment.erb` partial here, but I left this code alone for now. It is not much changed from the code extracted from `service/client/method/docs/sample.erb`.